### ISSUE TITLE
Implement the Galactic absorption in the simulation

### ISSUE THREAD

**Issue by lucabaldini** [lucabaldini/ximpol#120]
And cross check our results against those of the sensitivity calculator.

**Comment by lucabaldini** [comment]
We might want to take a look at 
http://yt-project.org/doc/analyzing/analysis_modules/photon_simulator.html
and see how they interfaced the code to pyxspec (although I'd be reluctant to add pyxspec as a required dependency--but maybe we can use their approach to fill a bivariate spline that we can store and use).

**Comment by lucabaldini** [comment]
We shall also have a model for N_H as a function of the sky position, so that for actual sources we can automatically set the column density without the need of specifying it explicitly.

**Comment by lucabaldini** [comment]
Michela has committed and pushed a python module to calculate the galactic absorption

```
ximpol/srcmodel/gabs.py
```

along with its fellow test file

```
ximpol/test/test_absorption.py
```

and the necessary configuration files

```
ximpol/srcmodel/fits/h1_nh_LAB.fits
ximpol/srcmodel/fits/h1_nh_DL.fits
ximpol/srcmodel/ascii/XsecFitParam.txt
```

**Comment by lucabaldini** [comment]
And I committed the .rst file

```
doc/modules/srcmodel.gabs.rst
```

so that the docstrings make it to the online documentation.

**Comment by lucabaldini** [comment]
Ok, I did some code cleanup and modified Michela's code to return splines that can be readily used in the simulations.

All the source classes now accept a `column_density` argument in the constructor, but that is not used yet.

Getting there...

**Comment by lucabaldini** [comment]
This is implemented. We wrote a unit test for a systematic comparison with the online sensitivity calculator and the basic outcome is in the table below (MDPs are in 2--4, 4--6 and 6--8 keV).

`
n_H       Gamma  MDP ratios
1e+21   1.0          [ 0.98020506  0.98331863  0.97438146]
1e+21   2.0          [ 0.97770388  0.98567684  0.97553513]
1e+22   1.0          [ 0.98111766  0.98313497  0.97456441]
1e+22   2.0          [ 0.97993842  0.98656215  0.97772386]
1e+23   1.0          [ 0.98625438  0.98129114  0.97432284]
1e+23   2.0          [ 0.98262508  0.97978988  0.9739401 ]
`

Basically ximpol runs ~2% lower (most likely not related with the galactic absorption) but things seem to be in order.
